install_config/configuring_sdn.adoc

```diff
@@ -165,6 +159,51 @@ xref:expanding-the-service-network[Expanding the service network] for more
 information.
 ====
 
+Be sure to restart the API and master services for any changes to take effect.
+
+----
+master-restart api
+master-restart controllers
+----
+
+[IMPORTANT]
+====
+The pod network settings on nodes need to match what has been set in clusterNetworks at the master level. This can be done by modifying parameters in the `*networkConfig*` section of the appropriate xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration map]:
+----
+proxyArguments:
+  cluster-cidr:
+  - 10.128.0.0/12 <1>
+----
+<1> The cidr should encompass all the cluster network cidr ranges defined at the master level but not conflict with other IP ranges: nodes, services, etc.
+
+====
+
+Once the master services have been restarted the configuration needs to be propagated to the nodes. Therefore the node services and ovs pods need to be restarted as well. To avoid downtime this is best approached by following the steps defined in xref:../admin_guide/manage_nodes.adoc#admin-guide-manage-nodes[Managing Nodes] for each node or group of nodes at a time:
```

Comment:
"wrote atomic-openshift-node service" instead of just "services" to be specific. Kept "for each node or group of nodes at a time" as it is an important point for avoiding downtime